Hyperbolic_surface_triangulation_2/README.md

```diff
@@ -0,0 +1,3 @@
+This folder contains the package Hyperbolic_surface_triangulation_2.
```

Comment:
Remove this file.  When we generate a tarball for a CGAL distribution this file would be right at the root.